tests/integration/targets/aws_glue_job/tasks/main.yml

```diff
@@ -0,0 +1,258 @@
+---
+- name: run aws_glue_job integration tests
+  module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+      purge_tags: no
+
+  block:
+    
+    # integration test setup
+    - name: create aws glue iam role for integration tests
+      iam_role:
+        name: my-glue-iam-role
```

Comment:
Rather than hard-coding a name we should use `{{ resource_prefix }}-something`.

resource_prefix is automatically populated by the Ansible test rig, and is unique for each run, meaning that the tests can be run in parallel for multiple versions of ansible/python/whatever.

I would generally use "role_name" and "job_name" and define them in defaults/main.yml